docs/DeveloperGuide.md

```diff
@@ -362,32 +362,58 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Use case: UC4 - Delete Fosterers**
 
-**Preconditions:** A list of fosterers is currently displayed.
-
 **MSS**
 
-1. Foster Manager requests to delete fosterers referenced by their index shown in the list.
-2. System deletes selected fosterers.
-3. System displays the updated list of fosterers.
+1. Foster Manager requests to view a list of fosterers.
+2. Foster Manager requests to delete fosterers referenced by their index shown in the list.
+3. System deletes selected fosterers.
+4. System displays the updated list of fosterers.
 
     Use case ends.
 
 **Extensions**
 
-* 1a. The displayed list is empty. 
+* 2a. The displayed list is empty. 
+  * 2a1. System indicates error.
 
-    Use case ends.
+      Use case ends.
```

Comment:
you are right :O v strange because now my tab will automatically add in that extra space HMMM ill just copy paste the use case ends line from somewhere else, thanks!